code/modules/atmospherics/machinery/other/zvent.dm

```diff
@@ -1,3 +1,15 @@
+
+/*
+	Remie note:
+	I added proper multiZ, but this thing is ancient
+	I "updated" it, and it DOES work, but it's
+	not in RPDs, pipe dispensers etc and some of its
+	content (on/off, volume_rate) aren't implemented.
+
+	Use this for ruins and stuff I guess, or fix it up proper.
+*/
```

Comment:
sure I guess, it just seemed cruel to kill it when it survived, semi-working, for such a long time.
rip zvent.